tests/fixtures/simulation.yml

```diff
@@ -128,6 +128,14 @@
   contains:
     - "[Simulator] Running ./vendor/bin/drush -y --root=$(pwd)/build --site-name='Site name' --site-mail=info@example.org --locale=en --account-mail=admin@example.org --account-name=admin --account-pass=admin --sites-subdir=default --db-url='mysql://root:root@127.0.0.1:3306/drupal' site-install standard"
 
+- command: 'drupal:site-install --database-type=sqlite3'
+  configuration:
+    drupal:
+      post_install: []
+  composer: ''
+  contains:
+    - "[Simulator] Running ./vendor/bin/drush -y --root=$(pwd)/build --site-name='Site name' --site-mail=info@example.org --locale=en --account-mail=admin@example.org --account-name=admin --account-pass=admin --sites-subdir=default --db-url='sqlite3://root:root@127.0.0.1:3306/drupal' site-install standard"
```

Comment:
We should add an extra parameter in order to specify the sqllite file, see Drush documentation:

```
    /**
     * Install Drupal along with modules/themes/configuration/profile.
     *
     * @command site:install
     * @param $profile An install profile name. Defaults to 'standard' unless an install profile is marked as a distribution. 
...
     * @usage drush si --db-url=mysql://root:pass@localhost:port/dbname
     *   Install using the specified DB params.
     * @usage drush si --db-url=sqlite://sites/example.com/files/.ht.sqlite
     *   Install using SQLite
...
     */
```